### PR TITLE
:world_map: Adds massdrop/alt/favorable-mutation keymap

### PR DESCRIPTION
**Pull request by favorable-mutation**
## Description

Adds my personal keymap.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+void matrix_init_user(void) {
+};
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void) {
+};
+
+#define MODS_SHIFT  (get_mods() & MOD_BIT(KC_LSHIFT) || get_mods() & MOD_BIT(KC_RSHIFT))
+#define MODS_CTRL  (get_mods() & MOD_BIT(KC_LCTL) || get_mods() & MOD_BIT(KC_RCTRL))
+#define MODS_ALT  (get_mods() & MOD_BIT(KC_LALT) || get_mods() & MOD_BIT(KC_RALT))
```
Comment (fauxpark) [inline keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c]: You can actually just write `get_mods() & MOD_MASK_SHIFT` etc. here.

File: keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______  \
+    ),
+    /*
+    [X] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______, _______, \
+        _______, _______, _______,                            _______,                            _______, _______, _______, _______, _______  \
```
Comment (fauxpark) [inline keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c]: The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater without them.

File: keyboards/massdrop/alt/keymaps/favorable-mutation/config.h [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+// toggle one-shot layer keys after this many presses
+#define ONESHOT_TAP_TOGGLE 3
+
+// set time held before Mod-Tap key sends mod code
+#define TAPPING_TERM 1000
+
+// permissive hold: if holding down a Mod-Tap key while pressing another, it
+// acts as a mod, even within the tapping term
+#define PERMISSIVE_HOLD
```
Comment (fauxpark) [inline keyboards/massdrop/alt/keymaps/favorable-mutation/config.h]: Most of this file is identical to the keyboard level config.h, and should be removed. All you need here are these lines, and `#pragma once` at the top.

File: keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c (hunk 2 above)
Comment (favorable-mutation) [reply keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c]: Oooooo agreed, removing!

File: keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c (hunk 1 above)
Comment (favorable-mutation) [reply keyboards/massdrop/alt/keymaps/favorable-mutation/keymap.c]: Nice, changed.

File: keyboards/massdrop/alt/keymaps/favorable-mutation/config.h (hunk 3 above)
Comment (favorable-mutation) [reply keyboards/massdrop/alt/keymaps/favorable-mutation/config.h]: Did not realize it would inherit like that! Removed; thanks so much for reviewing!

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**